src/billing/reducers/mockBillingData.ts

```diff
@@ -62,11 +61,6 @@ export const mockPaymentMethods: PaymentMethod[] = [
 ]
 
 export const mockRegion: Region = {
-  title: 'EU Central',
-  isBeta: false,
-  isAvailable: true,
-  provider: 'aws',
-  region: 'eu-central',
```

Comment:
Rather than removing the mock data from the mocks, we should update the test to check for these property types